app/src/main/res/layout/reading_text_size_fragment.xml

```diff
@@ -67,8 +68,8 @@
 
           <TextView
             android:id="@+id/reading_text_size_small_textview"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="10dp"
+            android:layout_height="17dp"
```

Comment:
@Mehul-Bisht can you explain the need for this change

Reply:
@ARJUPTA seems that I was checking something and forgot to revert this change, I'll do that now.